src/test/resources/refaster/FindListAddRecipe.java

```diff
@@ -64,7 +64,6 @@ public String getDescription() {
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         JavaVisitor<ExecutionContext> javaVisitor = new AbstractRefasterJavaVisitor() {
             JavaTemplate before;
-            JavaTemplate after;
```

Comment:
This one and below never had an `after` template method to begin with; the recipes aim to find only.